site/src/component/Review/Review.tsx

```diff
@@ -42,47 +42,22 @@ const Review: FC<ReviewProps> = (props) => {
                     let bScore = b.score + (b.verified ? 10000 : 0);
                     return bScore - aScore;
                 })
-                let reviewIDs = [];
-                for(let i = 0;i<data.length;i++){
-                    reviewIDs.push(data[i]._id);
-                }
                 const req = {
-                    ids: reviewIDs as string[]
+                    ids: data.map(review => review._id!)
                 }
                 let colors = await getColors(req);
+                console.log(colors);
                 setVoteColors(colors);
                 dispatch(setReviews(data));
             });
     }
 
-    const updateVoteColors = async () => {
-        let reviewIDs = [];
-        for(let i = 0;i<reviewData.length;i++){
-            reviewIDs.push(reviewData[i]._id);
-        }
-        const req = {
-            ids: reviewIDs as string[]
-        }
-        let colors = await getColors(req);
-        setVoteColors(colors);
-    }
-
-    const getU = (id: string | undefined) => {
+    const getVoteColor = (id: string | undefined) => {
         let temp = voteColors as Object;
         let v = (temp[id as keyof typeof temp]) as unknown as number;
-        if(v == 1){
-            return {
-                colors: [true, false]
-            }
-        }else if(v == -1){
-            return {
-                colors: [false, true]
-            }
-        }
-        return {
-            colors: [false, false]
-        }
+        return v ?? 0;
```

Comment:
Could you add some comments explaining what this does and change the variable names to be more descriptive? Why is voteColors being cast as an Object? Can we change the type its declared with to match what we're getting from the backend?

Reply:
Moving this to #401 since it is an existing issue and is kind of out of scope of this PR.